server/src/native-test/java/io/whitefox/api/deltasharing/server/DeltaSharesApiImplIT.java

```diff
@@ -3,15 +3,15 @@
 import static io.restassured.RestAssured.given;
 import static org.hamcrest.Matchers.*;
 
-import io.quarkus.test.junit.QuarkusIntegrationTest;
+import io.quarkus.test.junit.QuarkusTest;
 import io.whitefox.OpenApiValidationFilter;
 import io.whitefox.api.deltasharing.encoders.DeltaPageTokenEncoder;
 import io.whitefox.core.services.ContentAndToken;
 import jakarta.ws.rs.core.Response;
 import java.nio.file.Paths;
 import org.junit.jupiter.api.Test;
 
-@QuarkusIntegrationTest
+@QuarkusTest
```

Comment:
Why this change?

Reply:
this is because this native test are fails during build.

Reply:
Well, this is not the way to solve it 😛